internal/handler/ice.go

```diff
@@ -117,6 +117,14 @@ func (h *Handler) getIceServerConfForStunnerConf(params types.GetIceAuthParams,
 		h.log.Debugf("Considering Listener: namespace: %s, gateway: %s, listener: %s", namespace,
 			gateway, listener)
 
+		if params.PublicIp != nil {
```

Comment:
Again, let's just swap the precedence order here

Reply:
In this case I believe the code is correct: if the `public-ip` parameter is present, we use it. Otherwise we look at the environment setting. So if both are set, the URL parameter takes precedence. Is this what we want?

Reply:
Let's make this `params.PublicAddr`, shall we?